src/main/kotlin/com/aroundme/likes/LikeService.kt

```diff
@@ -37,4 +37,19 @@ class LikeService (
         logger.info("Service - Saving like to $contentId by $userId successfully: $savedLikes")
     }
 
+    /**
+     * Decreases a like count on content by user
+     *
+     * @param contentId, userId
+     * @return void
+     */
+    @Transactional
+    fun decreaseLikeCount(contentId: Long, userId: Long) {
```

Comment:
As discussed in offline, `decreaseLikeCount` is not actually decreasing its count even if that function's context is to decrease its count.  How about rename it to delete a related row?  Actual behavior is to delete a matched row with contentId and user Id.